modules/openapi-generator/src/main/resources/C-libcurl/model-header.mustache

```diff
@@ -11,6 +11,11 @@
 #include "../external/cJSON.h"
 #include "../include/list.h"
 #include "../include/keyValuePair.h"
+#include "../include/binary.h"
+
+char *base64encode(const void *b64_encode_this, int encode_this_many_bytes);
+
+char *base64decode(const void *b64_decode_this, int decode_this_many_bytes, int *decoded_bytes);
 
```

Comment:
The two functions ```base64encode``` or ```base64decode``` are declarated in apiClient.h
So why they are needed here ?

Reply:
Because I don't `#include apiClient.h` everywhere. I am actually considering to move base64* to binary.h / binary.c

Reply:
OK. I got it.
But users may want to use these 2 functions , but they do not want to include "binary.h" 
So I suggest keeping the definition and decalration of the base64* functions in apiClient.h|c, but add the declarations to binary.h

Reply:
ok

Reply:
Moving to binary.h/binary.c is also acceptable for me.

Reply:
@michelealbano you dont need to include apiClient.h any where. You just need to import the main api's api.h this will import all the models required and supporting files for that API

Reply:
Thinking about it, I will move them to binary.h/binary.c. I `#include "../include/binary.h"` in apiClient.h, thus the functions are available to whoever includes apiClient.h.